keystorage/aws/src/main/java/tech/pegasys/signers/aws/AwsSecretsManager.java

```diff
@@ -68,6 +76,39 @@ static AwsSecretsManager createAwsSecretsManager() {
     }
   }
 
+  public <R> Collection<R> mapSecretsList(
+      final String prefix, final BiFunction<String, String, R> mapper) {
+    final Set<R> result = ConcurrentHashMap.newKeySet();
+    secretsManagerClient.listSecretsPaginator().stream()
+        .parallel()
+        .forEach(
+            p ->
+                p.secretList()
+                    .parallelStream()
+                    .map(secretEntry -> secretEntry.name())
+                    .filter(secretName -> secretName.startsWith(prefix))
```

Comment:
This seems like an odd filter to have. What would make sense from how AWS is used? I think that tag might be more appropriate. But also supporting having no filter at all.

Reply:
The current signers read-only IAM policy used for the unit tests have restricted permissions and can only read secrets that have a resource/ARN under `arn:aws:secretsmanager:*:*:secret:signers-aws-integration/*`.
This appears to be the common approach for grouping secrets for specific use cases, hence the filter implemented in this method.

Reply:
I agree that tags would be more appropriate